charts/csi-vxflexos/values.yaml

```diff
@@ -10,21 +10,33 @@ version: v2.11.0
 
 images:
   # "driver" defines the container image, used for the driver container.
-  driver: dellemc/csi-vxflexos:v2.11.0
+  driver:
+    image: dellemc/csi-vxflexos:v2.11.0
```

Comment:
Would dependabot pick it up if the image were `driverImage` or `powerflexSdcImage`?

Reply:
no, it does not